File: src/test/resources/org/jetbrains/research/ml/dataset/anonymizer/transformation/java/anonymization/data/classes_and_methods/in_2_overriding.java
public class Test2_0 implements Operationable {

    @Override
    public boolean equals(Object obj) {
        return super.equals(obj);
    }

    public static void main() {
    }

    @Override
    public int calculate(int x, int y) {
        return 0;
    }
}

class Test2_1 extends Test2_0 {

}

interface Operationable {
    int calculate(int x, int y);
}

File: src/test/resources/org/jetbrains/research/ml/dataset/anonymizer/transformations/anonymization/kotlin/data/classes_and_methods/in_2_overriding.kt
public class Test2_0 : Operationable {

    override fun equals(other: Any?): Boolean {
        return super.equals(other);
    }

    fun main() {

    }


    override fun calculate(x: Int, y: Int): Int {
        return 0;
    }
}

class Test2_1 : Test2_0 {

}

interface Operationable {
    fun calculate(x: Int, y: Int): Int
}
